packages/trader/src/Modules/CFD/Components/cfd-demo-account-display.jsx

```diff
@@ -181,7 +181,7 @@ const CFDDemoAccountDisplay = ({
                             [localize('Leverage')]: localize('Up to 1:100'),
                             [localize('Margin call')]: localize('100%'),
                             [localize('Stop out level')]: localize('50%'),
-                            [localize('Number of assets')]: localize('50+'),
+                            [localize('Number of assets')]: localize('70+'),
```

Comment:
Can we move the specs to `cfd-specifications.js` file?

Reply:
Yes, done.